### ISSUE TITLE
Missing Script: _zapier-build

### ISSUE THREAD

**Issue by ThomasGl** [zapier/zapier-platform#268]
### Bug Description

Confusing and recurring bugs on the action of running "zapier push" or "zapier build"

both return the same error. An attachment of the error and the log is attached.

when run zapier build:

![zapier build error](https://user-images.githubusercontent.com/56161967/91973231-92757c00-ed24-11ea-8aba-20f5f3170fea.PNG)

when run zapir build --debug:

![zapier build error debug](https://user-images.githubusercontent.com/56161967/91973261-a02b0180-ed24-11ea-8bc9-e82fcdc67f00.PNG)

log file:

![zapier error log](https://user-images.githubusercontent.com/56161967/91973294-ae791d80-ed24-11ea-9887-43a4d62327a4.PNG)

### Reproduction Steps

1. run : "zapier push" or "zapier build"

### Version Info

- Version info:  10.1.0
- Operating System: Windows 10
- App id:  115315

**Comment by eliangcs** [comment]
Thanks for reporting! This will be fixed in v10.1.1 by #265. Meanwhile, as a workaround, you can either:

* Downgrade back to CLI v10.1.0
* Add a no-op `_zapier-build` to your `package.json` as such:

    ```json
    {
      "scripts": {
        "_zapier-build": ""
      }
    }
    ```

**Comment by xavdid** [comment]
a fix has been released as Zapier CLI 10.1.1, available now!
